shared/editor/nodes/CodeFence.ts

```diff
@@ -283,7 +285,23 @@ export default class CodeFence extends Node {
   };
 
   get plugins() {
-    return [Prism({ name: this.name }), Mermaid({ name: this.name })];
+    return [
+      Prism({ name: this.name }),
+      Mermaid({ name: this.name }),
+      new Plugin({
+        key: new PluginKey("triple-click"),
+        props: {
+          handleDOMEvents: {
+            mousedown(view, event) {
+              const {
```

Comment:
This is great, works really well – these plugin dom event handlers are actually document-wide in scope though so it's probably best to check that the current selection is in a code block here.

Reply:
That's awesome! 

I was looking for a way to make sure selection is in code block, then I remembered you [mentioned that before](https://github.com/outline/outline/pull/3775#issuecomment-1182151456), and voila 

https://github.com/outline/outline/blob/9afda0a97125fa29aaa1301bc54363797a1b2a49/shared/editor/nodes/CodeFence.ts#L184 

there it was!